### ISSUE TITLE
Some doubts about the absolute value of ViCLIP similarity

### ISSUE THREAD

**Issue by LiuHuijie6410** [OpenGVLab/InternVideo#175]
Thanks for such beautiful work！
In the past, video-text similarity was usually calculated by calculating the similarity between each frame and text and then taking the average. If the text and video are aligned, the value calculated in this way is usually above 0.9.
However, The value calculated using ViCLIP is only about 0.3.
Could you explain the reason?

**Comment by zmy1116** [comment]
Yeah I want to ask this too, not sure how should we threshold out good matches given that cosine similarities are almost always lie between 0.2 -0.4.  I can see that `softmax(100* score)` is used to get relative closeness among a set of candidates but this doesn't help to exclude unmatched candidates .

**Comment by leemengtw** [comment]
Hi folks, I'm encountering the same issue and am just wondering did you guys figure out this or found a way to threshold out irrelevant candidates? 

Apologize for the direct mentions @shepnerd  @leexinhao but I think once this is clear, the InternVideo models would definitely get far more adaptions from the community so wondering can you guys can help us on this, thanks in advance!

**Comment by lq826311756** [comment]
Hi there, I'm facing the same issue: the similarity scores for text-visual sample matching consistently fall within the range of 0.2 to 0.3. Has anyone managed to solve this problem?
